### ISSUE TITLE
`Get-ParameterInfo` misses `IsDynamic` parameters on aliases

### ISSUE THREAD

**Issue by ninmonkey** [jdhitsolutions/PSScriptTools#101]
**Describe the bug**
`Get-ParameterInfo` misses `IsDynamic` parameters when using the alias name.

**To Reproduce**
```powershell
PS> Get-ParameterInfo gc | ft Name, Aliases, IsDynamic, Type


Name        Aliases    IsDynamic Type
----        -------    --------- ----
ReadCount                  False System.Int64
TotalCount  First,Head     False System.Int64
Tail        Last           False System.Int32
Path                       False System.String[]
LiteralPath PSPath,LP      False System.String[]
Filter                     False System.String
Include                    False System.String[]
Exclude                    False System.String[]
Force                      False System.Management.Automation.SwitchParameter
Credential                 False System.Management.Automation.PSCredential
```

**Expected behavior**

```powershell
PS> Get-ParameterInfo Get-Content | ft Name, Aliases, IsDynamic, Type


Name         Aliases    IsDynamic Type
----         -------    --------- ----
ReadCount                   False System.Int64
TotalCount   First,Head     False System.Int64
Tail         Last           False System.Int32
Path                        False System.String[]
LiteralPath  PSPath,LP      False System.String[]
Filter                      False System.String
Include                     False System.String[]
Exclude                     False System.String[]
Force                       False System.Management.Automation.SwitchParameter
Credential                  False System.Management.Automation.PSCredential
Delimiter                    True System.String
Wait                         True System.Management.Automation.SwitchParameter
Raw                          True System.Management.Automation.SwitchParameter
Encoding                     True System.Text.Encoding
AsByteStream                 True System.Management.Automation.SwitchParameter
Stream                       True System.String
```

**Desktop (please complete the following information):**
 - win10: 2004
 - Module Version: 2.31.0
 - PowerShell Version: 7.1.0

**Comment by jdhitsolutions** [comment]
That is an interesting bug.

**Comment by jdhitsolutions** [comment]
Not too difficult to fix. Working on a new release.  Of course, the workaround, for now, is to use the command name and not an alias.

**Comment by jdhitsolutions** [comment]
This should be fixed in v2.34.1.
